src/lf_apply.rs

```diff
@@ -161,29 +161,21 @@ pub(crate) unsafe fn rav1d_copy_lpf<BD: BitDepth>(
     let y_span = lr_plane_sz[0] as isize - y_stride;
```

Comment:
I think you should be able to remove `y_span` and `uv_span` here as well. They were used to compute the base address of the slice. It seems superfluous to subtract `y_span` in `yuv_offset[0]` and add it back in `dst_offset[0]`